### PR TITLE
Raid patches

### PR DESCRIPTION
**Pull request by otooleam**

### CODE REVIEW

File: PokeStar/PokeStar/Modules/RaidCommands.cs [hunk 1]
```diff
@@ -364,7 +374,10 @@ private static Embed BuildPlayerInviteEmbed(ImmutableList<SocketGuildUser> invit
       {
          StringBuilder sb = new StringBuilder();
          for (int i = 0; i < invite.Count; i++)
+         {
             sb.AppendLine($"{raidEmojis[i]} {(invite.ElementAt(i).Nickname == null ? invite.ElementAt(i).Username : invite.ElementAt(i).Nickname)}");
+            sb.AppendLine($"{cancelEmoji} Cancel");
```
Comment (Marthyu-Verat) [inline PokeStar/PokeStar/Modules/RaidCommands.cs]: this will add a cancel for every person

Comment (otooleam) [reply PokeStar/PokeStar/Modules/RaidCommands.cs]: grrr

File: PokeStar/PokeStar/DataModels/Raid.cs [hunk 2]
```diff
@@ -122,33 +125,52 @@ public bool PlayerAdd(SocketGuildUser player, int partySize, SocketGuildUser inv
...
          else
          {
-            foreach (var group in Groups)
+            for (int i = 0; i < Groups.Count; i++)
             {
+               RaidGroup group = Groups.ElementAt(i);
                if (group.HasPlayer(player))
                {
+                  bool everyoneWasReady;
+                  if (group.HasPlayer(player))
```
Comment (Marthyu-Verat) [inline PokeStar/PokeStar/DataModels/Raid.cs]: switch to bool everyoneWasReady = group.HasPlayer(player);

File: PokeStar/PokeStar/DataModels/Raid.cs [hunk 3]
```diff
@@ -122,33 +125,52 @@ public bool PlayerAdd(SocketGuildUser player, int partySize, SocketGuildUser inv
...
                return false;
          }
-         var newGroup = Groups.ElementAt(group).SplitGroup();
-         if (newGroup != null)
-            Groups.Add(newGroup);
-         CheckMergeGroups();
-         return true;
+
+         RaidGroup newGroup;
+         if (Groups.Count < GROUP_LIMIT)
```
Comment (Marthyu-Verat) [inline PokeStar/PokeStar/DataModels/Raid.cs:130]: So what happens to the person who was added now that the group is over 20 people?

**Pull request merged**